src/osx/Installer.Mac/layout.sh

```diff
@@ -27,7 +27,21 @@ GITLAB_UI_SRC="$SRC/shared/GitLab.UI.Avalonia"
 
 # Build parameters
 FRAMEWORK=net6.0
-RUNTIME=osx-arm64
+
+TEST_ARCH=`uname -m`
+case $TEST_ARCH in
+    "x86_64")
+        RUNTIME="osx-x64"
+        ;;
+    "arm64")
+        RUNTIME="osx-arm64"
```

Comment:
Verify this is a [supported RID](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#macos-rids). Also, although it would be ideal to publish a [Universal binary](https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary), dotnet doesn't currently support this (I found a [thread](https://docs.microsoft.com/en-us/answers/questions/471880/net-core-support-in-new-mac-os-arm-based-processor.html) that suggests it _might_ be coming with .NET 7). So we'll probably want to build _and publish_ two applications - one for Intel Macs and one for M1 Macs.

Reply:
See above comment. I was disappointed that universal binaries were not supported yet in dotnet, I'll keep an eye out once this PR gets merged and update that. I did want to add cross-compiling that way those releases can be published like https://github.com/GitCredentialManager/git-credential-manager/pull/237 has, but I decided to hold off until I got some feedback.